metasrv/src/meta_service/raftmeta.rs

```diff
@@ -412,6 +412,47 @@ impl MetaNode {
         Ok(mn)
     }
 
+    pub async fn join_cluster(&self, conf: &RaftConfig) -> MetaResult<()> {
+        if conf.join.is_empty() {
+            return Ok(());
```

Comment:
Should we log a warning message when this config is empty?

Reply:
I think the elaborated name of this method should be `try_to_join_cluster_if_cli_arg_join_is_specified()`.

`conf.join` being empty means the user did not pass `--join` arguments to metasrv. It's totally OK.